warehouse/dbt_project.yml

```diff
@@ -22,6 +22,7 @@ clean-targets:         # directories to be removed by `dbt clean`
   - "dbt_packages"
 
 vars:
+  FULL_REFRESH_SCHEDULE_START: '2021-04-16'
```

Comment:
nitpick: should this variable name say `GTFS_SCHEDULE`? just thinking that the word "schedule" for something like "full refresh" could maybe benefit from full context idk